db/migrate/20170208002441_create_pins.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePins < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pins do |t|
+      t.text :description
+      t.text :image_url
```

Comment:
I imagine this will change when we add Paperclip or something else. We will need something here for testing for now.